python/mujincontrollerclient/realtimerobotclient.py

```diff
@@ -133,23 +179,23 @@ def MoveJointStraight(self, deltagoalvalue, jointName, timeout=10, robotspeed=No
     def MoveToolLinear(self, goaltype, goals, toolname=None, timeout=10, robotspeed=None, **kwargs):
         """Moves the tool linear
 
-        :param goaltype: Type of the goal, e.g. translationdirection5d
-        :param goals: Flat list of goals, e.g. two 5D ik goals: [380,450,50,0,0,1, 380,450,50,0,0,-1]
-        :param toolname: Name of the manipulator, default is self.toolname
-
-        :param maxdeviationangle: How much the tool tip can rotationally deviate from the linear path
-        :param plannername:
-
-        :param workspeed: [anglespeed, transspeed] in deg/s and mm/s
-        :param workaccel: [angleaccel, transaccel] in deg/s^2 and mm/s^2
-        :param worksteplength: Discretization for planning MoveHandStraight, in seconds.
-        :param workminimumcompletetime: Set to trajduration - 0.016s. EMU_MUJIN example requires at least this much
-        :param workminimumcompleteratio: In case the duration of the trajectory is now known, can specify in terms of [0,1]. 1 is complete everything
-        :param numspeedcandidates: If speed/accel are not specified, the number of candiates to consider
-        :param workignorefirstcollisionee: time, necessary in case initial is in collision, has to be multiples of step length?
-        :param workignorelastcollisionee: time, necessary in case goal is in collision, has to be multiples of step length?
-        :param workignorefirstcollision:
-
+        Args:
+            goaltype (str): Type of the goal, e.g. translationdirection5d
+            goals (list[float]): Flat list of goals, e.g. two 5D ik goals: [380,450,50,0,0,1, 380,450,50,0,0,-1]
+            toolname (str, optional): Name of the manipulator. Default: self.toolname
+            timeout (float, optional):  (Default: 10)
+            robotspeed (float, optional):
+            workmaxdeviationangle (float): How much the tool tip can rotationally deviate from the linear path. In deg.
```

Comment:
This is the only actual change. The parameter was outdated, as discussed [here](https://mattermost.mujin.co.jp/engineering/pl/nmx4twri6f8ipy8fb7s3xz193e).